### ISSUE TITLE
Allow players to change their account name

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#70]
People clearly do want it.

![image](https://user-images.githubusercontent.com/87503574/189524309-271030f4-f05f-43d2-a837-5ce120261496.png)

I think it should cost gems, not coins. Most io games lock this feature behind premium currency, primarily to prevent abuse. We don't want players changing their account name daily. Gems can still be obtained reasonably easily every time a new season arrives, but the opportunity cost becomes higher (coins are comparably disposable)

**Comment by DisposableKatyusha** [comment]
It won't hurt to add. I am for this.

**Comment by snipergamingop** [comment]
Yes please. I would love to see this added.

**Comment by Gregory-Y** [comment]
I agree that this is in fact and in opinion an epic suggestion. Since we all seem to agree imma add the discussion consensus label here.
